architectures/github/ios-pwa-app/template.yaml

```diff
@@ -0,0 +1,203 @@
+  annotations:
+    backstage.io/adr-location: https://github.com/99x/product-central-architectures/tree/main/architectures/docs/decisions/ios-pwa-app
+    backstage.io/techdocs-ref: url:https://github.com/99x/product-central-architectures/tree/main/architectures/docs/techdocs/ios-pwa-app
+  tags:
+    - pwa
+    - native-ios
+    - xcode
+    - ios-swift
+    - reference-architecture
+  links:
+    - url: https://github.com/99x/product-central-architectures
+      title: 99x Product Central Architectures
+      icon: dashboard
+
+spec:
+  owner: user:99xadmin
+  type: service
+
+  parameters:
+    - title: Provide information about the component
+      required:
+        - name
+        - repoUrl
+        - description
+      properties:
+        name:
+          title: Name
+          type: string
+          description: Unique name of the component
+          ui:autofocus: true
+          maxLength: 50
+          pattern: "^[a-zA-Z0-9-]*$"
+          ui:help: "Hint: Name should only contain alphanumeric characters and hyphens"
+          ui:options:
+            rows: 5
+        repoUrl:
+          title: Repository Location
+          type: string
+          ui:field: RepoUrlPicker
+          ui:options:
+            allowedHosts:
+              - github.com
+        description:
+          title: Description
+          type: string
+          description: A brief description of the component and its purpose
+          ui:options:
+            rows: 5
+
+    - title: Provide information about applications
+      required:
+        - coreAppName
+        - remoteAppName
+        - serverAppName
+      properties:
+        coreAppName:
+          title: What would you like to name the client core application?
+          type: string
+          description: The name of the client core application
+          maxLength: 50
+          pattern: "^[a-zA-Z0-9-]*$"
+          ui:help: "Hint: Name should only contain alphanumeric characters and hyphens"
+          default: core
+        remoteAppName:
+          title: What would you like to name the first client remote application? (You can always add more remote applications later)
+          type: string
+          description: The name of the genesis client remote application
+          maxLength: 50
+          pattern: "^[a-zA-Z0-9-]*$"
+          ui:help: "Hint: Name should only contain alphanumeric characters and hyphens"
+          default: remote
+        serverAppName:
+          title: What would you like to name the server application?
+          type: string
+          description: The name of the server application
+          maxLength: 50
+          pattern: "^[a-zA-Z0-9-]*$"
+          ui:help: "Hint: Name should only contain alphanumeric characters and hyphens"
+          default: server
+
+    - title: Provide information about the deployment environment and credentials
```

Comment:
Do you set deployment for the template to AWS? I dont see any deployment pipelines on your content